app/core/types/password/password_test.go

```diff
@@ -0,0 +1,29 @@
+package password_test
+
+import (
+	"stonehenge/app/core/types/password"
+	"testing"
+)
+
+const (
+	example = "Awd@@Awd2022"
+	wrong   = "Aws@@Aws2021"
+)
+
+var pass = password.From(example)
+
+func TestSuccessfulPasswordComparison(t *testing.T) {
```

Comment:
Uma sugestão: depois da uma olhada em [table tests](https://dev.to/boncheff/table-driven-unit-tests-in-go-407b), eu considero que fica mais conciso e com menos duplicação. 

No caso desse arquivo acho que não é problemático por conter apenas 2 testes, mas se houvesse mais casos ai eu acredito que o beneficio do table test era mais evidente

Reply:
Acho interessante tu fazer algum teste com esse padrão pra poder exercitar antes de pegar task pra valer